src/core/lib/iomgr/tcp_server_posix.cc

```diff
@@ -105,6 +136,100 @@ static grpc_error_handle tcp_server_create(grpc_closure* shutdown_complete,
   s->memory_quota = s->options.resource_quota->memory_quota();
   s->pre_allocated_fd = -1;
   gpr_atm_no_barrier_store(&s->next_pollset_to_assign, 0);
+  s->n_bind_ports = 0;
+  new (&s->listen_fd_to_index_map)
+      absl::flat_hash_map<int, std::tuple<int, int>>();
+  if (grpc_event_engine::experimental::UseEventEngineServer()) {
+    PosixEventEngineWithFdSupport::PosixAcceptCallback accept_cb =
```

Comment:
I'd suggest moving these 100 lines of code to their own EventEngine instantiation function.

Reply:
Moved it to a new function now.